index.bs

```diff
@@ -466,15 +521,13 @@ spec: webrtc-pc; urlPrefix: https://www.w3.org/TR/webrtc/#
           return a Promise rejected with a <code>TypeError</code>.
         </li>
         <li>
-          If <code>configuration.video</code> in non-null and
-          <code>configuration.video.contentType</code> is not a <a>valid
-          VideoConfiguration contentType</a>, return a Promise rejected with a
+          If <code>configuration.video</code> is <a>present</a> and is not a
```

Comment:
It looks like https://wicg.github.io/media-capabilities/#create-a-mediacapabilitiesinfo-algorithm is the place where the content type is handled again, with the wording "If the user agent is able to decode the media represented by configuration ..."

So, since this is all happens inside of a single API call (`decodingInfo()` or `encodingInfo()`) and the settings don't change, it actually doesn't matter much where the different things happen. Nothing further from me.